chcemvediet/static/main/css/custom.sass

```diff
@@ -96,12 +96,18 @@ a:hover, a:focus
   &:hover, &:focus
     color: darken(#4d4d4d, 20%)
   @include clearfix()
-  > .chv-casestudy-img
-      display: block
-      float: left
-      height: ($chv-baseline * 8)
-      width: ($chv-baseline * 8)
-      margin-right: $chv-baseline
+  > .img-container
+    position: relative
+    margin-right: $chv-baseline
+    > .chv-casestudy-img
+        display: block
+        float: left
+        height: ($chv-baseline * 8)
+        width: ($chv-baseline * 8)
+    > .overlay-img
+      position: absolute
+      top: 44%
```

Comment:
@martinmacko47 Tieto cisla budu platit iba pri velkosti `chv-icon-3x`

Reply:
Idealne by bolo, keby css `.chv-casestudy` nezaviselo od konkretnych pouzitych class v ramci ikonky. Malo by to fungovat aj ked v definicii itemu pouzijem ine veci.

Asi by bolo najlepsie celu ikonku spravit cez separatnu classu ako popisujem v https://github.com/martinmacko47/chcemvediet/pull/407#discussion_r652030538